TODO/JS Exercises mix/Third_greatests_number.js

```diff
@@ -0,0 +1,13 @@
+let array = [1, 2, 3, 4, 5];
+
```

Comment:
imagine that there are more than 5 items inside your array, you need a solution that can be extensible (considering the performance) no matter the length of the array.

Reply:
@BZunino